pkg/stanza/fileconsumer/matcher/internal/filter/finder.go

```diff
@@ -6,8 +6,39 @@ package filter // import "github.com/open-telemetry/opentelemetry-collector-cont
 import (
 	"fmt"
 	"regexp"
+
+	"go.uber.org/multierr"
 )
 
+type Option interface {
+	// Returned error is for explanitory purposes only.
```

Comment:
If they are for explanation purposes, does that mean they _should_ be errors?

Should these be logs instead?

Reply:
They end up as logs, but we don't have a logger here. The intention is to return them as errors here and log the error at the higher level.